fib-prime.js

```diff
@@ -1,34 +1,35 @@
-var ispnum = function(num) {
-  for(var i = 2; i < num; i++)
-    if  (num % i === 0) return false;
-  return num > 1;
-};
+const isPrime = num => {
```

Comment:
Renamed function for better readable.
Used ES6 Function Expression to prevent reassignment function.